webcompat/form.py

```diff
@@ -34,11 +34,15 @@
 
 problem_choices = [
     (u'detection_bug', u'Desktop site instead of mobile site'),
-    (u'mobile_site_bug', u'Mobile site is not usable'),
-    (u'video_bug', u'Video doesn\'t play'),
-    (u'layout_bug', u'Layout is messed up'),
-    (u'text_bug', u'Text is not visible'),
-    (u'unknown_bug', u'Something else - I\'ll add details below')
+    (u'mobile_site_bug', u'Site is not usable'),
+    (u'layout_bug', u'Design is broken'),
+    (u'video_bug', u'Video or audio doesn\'t play'),
```

Comment:
Oh... oops. >_< You're right:

http://searchfox.org/mozilla-central/source/browser/base/content/browser-media.js#350

We have a couple of options here... 

1. Write a patch against Gecko and update to `media_bug` (and update the tests as well), and ensure we support both `video_bug` as a legacy alias to`media_bug`. We probably wouldn't have to maintain the alias for very long -- this is Nightly-only code and that population updates pretty quickly.

2. Just use `video_bug` and add a comment saying we use `video_bug` for legacy reasons, even though it might be used for audio bugs.

3. Do 2, but file a bug to come back and do 1 after the form rewrite lands.

3 is probably my preference. I can help you write your first Firefox patches if that's something that interests you.